api/src/main/resources/db/changelog/db.changelog-fees-managed-v5.yaml

```diff
@@ -163,3 +163,15 @@ databaseChangeLog:
           sql: >
             INSERT INTO service_type (name,creation_time,last_updated)
             VALUES ('other',current_timestamp,current_timestamp) ON CONFLICT DO NOTHING
+
+- changeSet:
+      id: 1534341462258
+      author: tarun
+      changes:
+      - addColumn:
+            columns:
+            - column:
+                  name: keyword
```

Comment:
Add an index on keyword and add another index to include all six refrence data in same index if possible